extensions/amp-carousel/0.1/slidescroll.js

```diff
@@ -16,6 +16,13 @@
 import {BaseCarousel} from './base-carousel';
 import {Layout} from '../../../src/layout';
 
+/** @const */
+const DIR_FORWARD = 1;
```

Comment:
1/-1 are very common constants for forward/reverse. IMHO, no sub-constanting needed.